src/Components/Shifting/ListFilter.tsx

```diff
@@ -11,7 +11,6 @@ import {
 import moment from "moment";
 import { getAnyFacility, getUserList } from "../../Redux/actions";
 import { useDispatch } from "react-redux";
-import { CircularProgress } from "@material-ui/core";
 import { SHIFTING_CHOICES } from "../../Common/constants";
 import { DateRangePicker, getDate } from "../Common/DateRangePicker";
```

Comment:
@ayushjnv1 this DatePicker indirectly uses Material UI components. You may make use of `DateRangeFormField` as a replacement which is the new custom components.

```suggestion
import { DateRangePicker, getDate } from "../Common/DateRangePicker";
```